build.gradle

```diff
@@ -60,6 +60,8 @@ allprojects {
 
     gradle.projectsEvaluated {
         tasks.withType(JavaCompile) {
+            sourceCompatibility = "8"
+            targetCompatibility = "8"
             options.compilerArgs << "-Xlint:unchecked" << "-Xlint:deprecation"
```

Comment:
@fpj  I guess this line is here for the same reason that we need to add source/targetCompatibilty at this point
this way we are sure that such options are applied

I took inspiration from this post:
https://stackoverflow.com/questions/21028438/gradle-sourcecompatibility-has-no-effect-to-subprojects